hal/buildroot/cvm-monitor.sh

```
@@ -0,0 +1,267 @@
+    done
+}
+
+function stop_cvm() {
+    log_message "Stopping TDX CVM and monitor..."
+    
+    # Stop monitor process
+    pkill -f "cvm-monitor.sh" 2>/dev/null
+    
+    # Stop CVM using PID file if it exists
+    if [ -f "$PIDFILE" ]; then
+        local pid=$(cat "$PIDFILE" 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)